apps/dotcms-block-editor/src/app/dot-block-editor/dot-block-editor.component.scss

```diff
@@ -0,0 +1,23 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,300;0,400;0,500;0,700;1,300;1,400&display=swap');
+@import url('https://fonts.googleapis.com/icon?family=Material+Icons');
+
+dotcms-block-editor {
+    display: block;
+    height: calc(100vh - 2rem); // 1rem sb boddy padding;
+    width: calc(100vw - 2rem); // 1rem sb boddy padding;
```

Comment:
We can't do this, because it breaks in some cases, can we just do `width: 100%` and `min-height: 500px`?